standup/settings.py

```diff
@@ -144,7 +144,7 @@ def path(*paths):
 USE_TZ = True
 
 # security
-
+SECURE_CONTENT_TYPE_NOSNIFF = config('SECURE_CONTENT_TYPE_NOSNIFF', default='true', parser=bool)
 SECURE_BROWSER_XSS_FILTER = config('SECURE_BROWSER_XSS_FILTER', default='true', parser=bool)
```

Comment:
Out of curiosity, why would we ever want these False? Why make them configurable?